packages/site-admin/src/__core_unstable_copy__/packages/edit-site/src/components/page/header.tsx

```diff
@@ -0,0 +1,53 @@
+/**
```

Comment:
If we decide to go with this "copy from Gutenberg" approach, I'd expect we keep a meticulous documentation, including wha, where, and from which hash/version we copied, plus a thorough changelog of all changes so we keep track of any potential divergence from core in the future.

Reply:
I couldn't agree more. I'd like to edit all commits or create a new branch from scratch to register all changes properly.